.github/workflows/pr-opened.yml

```diff
@@ -0,0 +1,36 @@
+name: on-pr-opened
+on:
+  pull_request:
+
+jobs:
+  lint-commits:
+    name: lint-commits
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
I don't think we need fetch depth here 🤔 ❓

Reply:
It's copied from `foundation`… 📋

Reply:
Okay, still I don't think it's required for this use case. Wanna try it ?

Reply:
Confirmed, we need it otherwise the workflow fails